README.md

```
@@ -0,0 +1,426 @@
+  - [Core Services & Responsibilities](#core-services--responsibilities)
+  - [Technical Workflow](#technical-workflow)
+- [Dataset](#dataset)
+  - [Key Features](#key-features)
+  - [Data Structure](#data-structure)
+  - [Generation Methodology](#generation-methodology)
+  - [Usage Notes](#usage-notes) 
+- [Installation](#installation)
+  - [Deploying with Helm](#deploying-with-helm)
+  - [Accessing the Services](#accessing-the-services)
+  - [Running Profiteer Locally](#running-profiteer-locally)
+  - [Airbyte Connection](#airbyte-connection)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)